### PR TITLE
Create feedback.md

### PR DESCRIPTION
**Pull request by hollyallen**
Some of my ideas on feedback.

### CODE REVIEW

File: people/feedback.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+Gather evidence for the feedback instead of relying on vague reports or hunches. Have concrete examples, ideally more than one for constructive feedback.
+
+### Behaviors
+Describe behaviors (is late to meetings) rather than traits or emotions (doesn't care about coworkers). You'll never know anyone else's thoughts, feelings, or motivations. You can observe behaviors in an objective and factual manner. This can be a hard habit change if you've built up years of giving feedback in the other way, but keep at it, it is worth switching.
+
+### Results
+When you describe a behavior, also say the result. This goes for positive and constructive feedback. "When you review pull requests right away, you really help the velocity of the whole team." "When you are regularly late for meetings, the effectiveness of the rest of the meeting is reduced because the meeting leader has to repeat information or wait for you."
+
+### Future Focus
+Don't well on the past, but focus on the future. The future can be altered, the past will never change.
```
Comment (sashax) [inline people/feedback.md]: I think you mean 'Don't dwell'

**Comment by sashax** [comment]
Apologies for nitpicking, but I'm not sure I agree with the usage of "constructive" where one might use "negative", especially since the first sentence appears to use the word differently.  Shouldn't all feedback be constructive, even the "keep doing what you're doing" stuff. Might the word "corrective" be better here?

**Comment by hollyallen** [comment]
Good suggestion @sashax . I took the "negative" suggestion, to enhance clarity. "Constructive feedback" is a normal term in management texts, but it is a wishy washy term.

**Comment by sashax** [comment]
LGTM!

**Comment by monfresh** [comment]
Is this worth linking to in the README?

**Comment by hollyallen** [comment]
Oh yeah @monfresh , happy to.

**Pull request merged**